src/components/PageSwitcher/components/EEGEduSpectra/EEGEduSpectra.js

```diff
@@ -97,52 +97,92 @@ export function setup(setData, Settings) {
 
 export function renderModule(channels) {
   function renderCharts() {
```

Comment:
find the max value over all spectra to make symetrical axis limits above and below zero (may not be worth it)